Doc/library/tempfile.rst

```diff
@@ -175,9 +175,9 @@ The module defines the following user-callable items:
    If you want to force a bytes return value with otherwise default behavior,
    pass ``suffix=b''``.
 
-   If *text* is specified, it indicates whether to open the file in binary
-   mode (the default) or text mode.  On some platforms, this makes no
-   difference.
+   If *text* is specified and ``True`` (or a truthy value), the file is
```

Comment:
I'd skip the mention of `True`, and just say "If *text* is specified and true, the file is" ...

I think the way it is is too much jargon for beginners. I quick glance through some of the other docs doesn't show any mention of `True` or truthiness, just "true".

Reply:
I concur.